### PR TITLE
add workflow to backfill packages to apk.cgr.dev

### PR DESCRIPTION
**Pull request by imjasonh**

### CODE REVIEW

File: .github/workflows/backfill.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                echo "Package already exists: ${pkg}"
+                continue
+              elif [ $code != "404" ]; then
+                echo "Unexpected response code: $code"
+                exit 1
+              fi
+
+              # Get package from GCS, continue if it doesn't exist (e.g., not in the repo)
+              echo "Downloading ${pkg} from GCS"
+              gsutil cp gs://wolfi-production-registry-destination/os/${arch}/${pkg} ${tmp}/ > /dev/null 2>&1 || continue
```
Comment (k4leung4) [inline .github/workflows/backfill.yaml:68]: nit: 
`gcloud alpha storage cp` claims to be faster
though i dont really care

**Review by luhring — DISMISSED** [review]

**Review by k4leung4 — DISMISSED** [review]

File: .github/workflows/backfill.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+              echo "Downloaded ${pkg} from GCS"
+
+              echo "Uploading ${pkg} to apk.cgr.dev"
+              curl --fail -X POST \
+                --user "user:${tok}" \
+                --data-binary "@${tmp}/${pkg}" \
```
Comment (mattmoor) [inline .github/workflows/backfill.yaml:74]: Switch to the flag that supports large files better?

Comment (imjasonh) [reply .github/workflows/backfill.yaml:74]: What flag is that again? (this is copied wholesale from the other repo, I should update it there too)

Comment (k4leung4) [reply .github/workflows/backfill.yaml:74]: https://github.com/curl/curl/issues/1385#issuecomment-291442642

use `-T <filename -X POST`?

**Review by k4leung4 — APPROVED** [review]

**Pull request merged**